transport/transport-runtime/src/main/AndroidManifest.xml

```diff
@@ -14,10 +14,16 @@
 
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.google.android.datatransport.runtime">
+  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <!--Although the *SdkVersion is captured in gradle build files, this is required for non gradle builds-->
     <!--<uses-sdk android:minSdkVersion="14"/>-->
 
     <application>
+        <service
+            android:name=".scheduling.jobscheduling.JobInfoSchedulerService"
+            android:label="JobScheduler Service"
+            android:permission="android.permission.BIND_JOB_SERVICE" >
```

Comment:
please add `android:exported="false"`, also it seems we don't need the label(none of our services/content-providers are labeled)